### ISSUE TITLE
Very slow to process a "Large" sized audio file. 

### ISSUE THREAD

**Issue by spobleteo** [chidiwilliams/buzz#350]
Very slow to process a "Large" sized audio file. Takes about 3 hours to transcribe a 1 hour long file. I have an app on my iPhone called Scribe and it performed the same task in 10 minutes. Why the difference? In any case, it is the first application I know for Windows that is WYSINWYG.

**Comment by spobleteo** [comment]
Hi. The app in the screen says: "Scribe", but in the App Store says: "Annual Notes Ultra (Annual)". 
![Scribe](https://user-images.githubusercontent.com/124691200/218205971-3a7b813a-c84d-49e0-b5d0-43eaa80cc2f0.jpeg)
Best

**Comment by Sircam19** [comment]
> Hi. The app in the screen says: "Scribe", but in the App Store says: "Annual Notes Ultra (Annual)".
> ![Scribe](https://user-images.githubusercontent.com/124691200/218205971-3a7b813a-c84d-49e0-b5d0-43eaa80cc2f0.jpeg)
> Best

Hey thanks for this... Interesting... I'll check it out, especially to see if the processing is done on a server. Merci Sir.

**Comment by abdcef** [comment]
> The app in the screen says: "Scribe", but in the App Store says: "Annual Notes Ultra (Annual)". 
> 

It's quite steeply priced. Is it worth the $99 subscription?

**Comment by spobleteo** [comment]
> 

It is expensive. It makes sense to pay if you are going to make heavy use of transcripts. There is an online application called: https://freesubtitles.ai/ Free to use: maximum file size 300 MB, maximum duration 1 hour.

**Comment by spobleteo** [comment]
> > 
> 
> It is expensive. It makes sense to pay if you are going to make heavy use of transcripts. There is an online application called: https://freesubtitles.ai/ Free to use: maximum file size 300 MB, maximum duration 1 hour.

About the other app for iPhone. The real name in the App Store is "Vienna Scribe". Best

**Comment by spobleteo** [comment]
> > 
> 
> It is expensive. It makes sense to pay if you are going to make heavy use of transcripts. There is an online application called: https://freesubtitles.ai/ Free to use: maximum file size 300 MB, maximum duration 1 hour.

About the other app for iPhone. The real name in the App Store is "Vienna Scribe"

**Comment by abdcef** [comment]
I hope we have more cost / resource effective alternatives that can use the large model!

**Comment by spobleteo** [comment]
Yes. Buy the velocity of the Scribe in almost instantaneous. Of course, Buzz es free, I wish someone do the same for iPhone or Android.

**Comment by Sircam19** [comment]
> Yes. Buy the velocity of the Scribe in almost instantaneous. Of course, Buzz es free, I wish someone do the same for iPhone or Android.

On mobile device for iphone there is Hello Transcribe. It used to be free (until a few days ago) but now $2 USD per month... Does not output SRT or VRT only plain text. Does not use servers so all data stays on device. Not sure if there is an android version. It's great if your in a bind and only have your iphone. For desktop transcription very much appreciative of all the heavy and involved work that Chidi has done.

**Comment by spobleteo** [comment]
> > Yes. Buy the velocity of the Scribe in almost instantaneous. Of course, Buzz es free, I wish someone do the same for iPhone or Android.
> 
> On mobile device for iphone there is Hello Transcribe. It used to be free (until a few days ago) but now $2 USD per month... Does not output SRT or VRT only plain text. Does not use servers so all data stays on device. Not sure if there is an android version. It's great if your in a bind and only have your iphone. For desktop transcription very much appreciative of all the heavy and involved work that Chidi has done.

I'll check it out! Thanks
